src/Microsoft.Framework.Configuration/ConfigurationBuilder.cs

```diff
@@ -7,18 +7,18 @@ namespace Microsoft.Framework.Configuration
 {
     public class ConfigurationBuilder : IConfigurationBuilder
     {
-        private readonly IList<IConfigurationSource> _sources = new List<IConfigurationSource>();
+        private readonly IList<IConfigurationProvider> _providers = new List<IConfigurationProvider>();
 
-        public ConfigurationBuilder(params IConfigurationSource[] sources)
-            : this(null, sources)
+        public ConfigurationBuilder(params IConfigurationProvider[] providers)
+            : this(null, providers)
         {
         }
 
-        public ConfigurationBuilder(string basePath, params IConfigurationSource[] sources)
+        public ConfigurationBuilder(string basePath, params IConfigurationProvider[] providers)
```

Comment:
`basePath` argument shouldn't be here anymore. I thought we had successfully removed in @kirthik's PR.